### PR TITLE
Points

### PR DESCRIPTION
**Pull request by almarklein**

### CODE REVIEW

File: visvis2/renderers/wgpu/pointsrender.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    aa_width = 1.0
+    d = distance(pcoord_pixels, vec2(hsize, hsize))
+    if d <= hsize - 0.5 * aa_width:
+        out_color = u_points.color.rgba  # noqa - shader output
+    elif d <= hsize + 0.5 * aa_width:
+        alpha = 0.5 + (hsize - d) / aa_width
+        alpha = alpha ** 2  # this works better
+        out_color = vec4(color.rgb, color.a * alpha)  # noqa - shader output
+    else:
+        return  # discard
```
Comment (almarklein) [inline visvis2/renderers/wgpu/pointsrender.py:50]: Look, if-statements and a builtin function to calculate the distance!

**Comment by Korijn** [comment]
Care to share some screenshots of both shaders in action?

File: visvis2/renderers/wgpu/pointsrender.py (hunk 1 above)
Comment (Korijn) [reply visvis2/renderers/wgpu/pointsrender.py:50]: Awesome!!

File: visvis2/materials/_points.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+
+
+class GaussianPointsMaterial(PointsMaterial):
+    """ A material for points, renders Gaussian blobs with a standard
+    deviation of 1/6 of the size.
+    """
+
+
+# idea: a MarkerMaterial with more options for the shape, and an edge around the shape.
+# Though perhaps such a material should be part of a higher level plotting lib.
```
Comment (Korijn) [inline visvis2/materials/_points.py:76]: If it's a generic material, I don't see why it couldn't be a part of this library. It isn't a big priority right now tho

**Comment by almarklein** [comment]
afbeelding
afbeelding

**Pull request merged**